src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/dft.cpp

```diff
@@ -27,39 +33,58 @@ using namespace LayerTestsDefinitions;
 
 INSTANTIATE_TEST_SUITE_P(smoke_DFT_2d,
                          DFTLayerTest,
-                         combine({{10, 2}, {1, 2}},  // input shapes
-                                 {{0}, {-1}},        // axes
-                                 {{}, {5}}),         // signal sizes
+                         combine({{10, 2}},   // input shapes
```

Comment:
Is there any specific reason that you reduced the tests?

Reply:
Yes, I have reduced input shape combinations because the second combination didn't make much difference, but it caused too many tests to generate and run. And about negative axes, I just separated them into the distinct test suite, so we test the same logic, but only in one test suite instead of all.